app/src/main/java/org/simple/clinic/AppDatabase.kt

```diff
@@ -77,4 +78,43 @@ abstract class AppDatabase : RoomDatabase() {
         """)
     }
   }
+
+  class Migration_4_5 : Migration(4, 5) {
+
+    override fun migrate(database: SupportSQLiteDatabase) {
+      // Update local searchable name in the Patient table to strip out the newly added characters
+      database.inTransaction {
+        compileStatement("""update "Patient" set "searchableName"=? where "uuid"=?""")
+            .use { statement ->
+              query("""select "uuid","fullName" from "Patient"""")
+                  .use { cursor ->
+                    val uuidIndex = cursor.getColumnIndex("uuid")
+                    val fullNameIndex = cursor.getColumnIndex("fullName")
+
+                    generateSequence { if (cursor.moveToNext()) cursor else null }
+                        .map { it.getString(uuidIndex) to it.getString(fullNameIndex) }
+                        .map { (uuid, fullName) -> uuid to convertNameToSearchableForm(fullName) }
```

Comment:
I don't have any strong opinion, but what do you think of merging these two maps into one?

```
.map { getString(uuidIndex) to convertNameToSearchableForm(getString(fullNameIndex)) }
```
https://blog.danlew.net/2018/02/20/when-you-dont-need-a-map/